pick_place_repeat_server:
  ros__parameters:
    object_names: ['table', 'table_gluider', 'inbeam_platform','holder_base_inbeam','holder_shaft_inbeam','holder_head_inbeam','holder_base_storage','holder_shaft_storage','holder_head_storage']
    objects:
      table:
        type: 'BOX'
        w: 2.0 # width
        d: 2.0 # depth
        h: 0.01 # height
        #w: 0.0 # pose, orientation 
        x: 0.0 # pose x
        y: 0.0 # pose y
        z: -0.03 # pose z
      table_gluider:
        type: 'BOX'
        w: 0.60 # width
        d: 0.15 # depth
        h: 0.02 # height
        #w: 0.0 # pose, orientation 
        x: 0.0 # pose x
        y: 0.0 # pose y
        z: -0.015 # pose z
      inbeam_platform:
        type: 'BOX'
        w: 0.30 # width
        d: 0.10 # depth
        h: 0.03 # height
        #w: 0.0 # pose, orientation 
        x: 0.0 # pose x
        y: 0.59 # pose y
        z: 0.355 # pose z 
      holder_base_inbeam:
        type: 'BOX'
        w: 0.06 # width
        d: 0.06 # depth
        h: 0.01 # height
        #w: 0.0 # pose, orientation 
        x: 0.006 # pose x
        y: 0.56 # pose y
        z: 10.475 # pose z 
      holder_shaft_inbeam:  #holder_shaft_inbeam
        type: 'BOX'
        w: 0.02 # width
        d: 0.01 # depth
        h: 0.04 # height
        #w: 0.0 # pose, orientation 
        x: 0.006 # pose x
        y: 0.525 # pose y
        z: 0.40 # pose z 
      holder_head_inbeam:
        type: 'BOX'
        w: 0.08 # width
        d: 0.01 # depth
        h: 0.02 # height
        #w: 0.0 # pose, orientation 
        x: 0.006 # pose x
        y: 0.55 # pose y
        z: 0.435 # pose z 
      holder_base_storage:
        type: 'BOX'
        w: 0.06 # width
        d: 0.06 # depth
        h: 0.01 # height
        #w: 0.0 # pose, orientation 
        x: -0.044 # pose x
        y: -0.51 #-0.55 # pose y
        z: 10.035 # pose z 
      holder_shaft_storage: #holder_shaft_torage
        type: 'BOX'
        w: 0.02 # width
        d: 0.01 # depth
        h: 0.04 # height
        #w: 0.0 # pose, orientation 
        x: -0.044 # pose x
        y: -0.53 #-0.51 # pose y
        z: 0.06 # pose z 
      holder_head_storage:
        type: 'BOX'
        w: 0.08 # width
        d: 0.01 # depth
        h: 0.02 # height
        #w: 0.0 # pose, orientation 
        x: -0.044 # pose x
        y: -0.51 # pose y
        z: 10.09 # pose z 
    home_pose: [-0.11536, -1.783732, 0.38816, -1.75492, 0.11484, 3.14159] #For real robot
    pickup_approach: [1.466, -2.042, -2.1293, -2.164, -0.105, 0.0]
    pickup_grasp: [1.099, -2.234, -1.728, -2.339, -0.489, -0.035]
    pickup_retreat: [1.099, -2.025, -1.571, -2.705, -0.489, -0.035]
    dropoff_approach: [2.618, -2.356, -1.763, -2.216, -2.215, 0.0]
    dropoff_grasp: [2.618, -2.566, -1.7453, -2.025, -2.217, 0.0]
    dropoff_retreat: [2.321, -2.496, -1.955, -1.902, -2.513, 0.0]
    joint_names: ['shoulder_pan_joint' , 'shoulder_lift_joint', 'elbow_joint', 'wrist_1_joint', 'wrist_2_joint', 'wrist_3_joint']1122
    pickup_pre_approach: [0.0, -2.2230397, -2.4030528 , -1.627018 , -1.541525, 3.141435] 
    pickup_pre_approach_stg_2: [-4.251453 , -2.2230678 , -2.4030139 , -1.6269628 , -1.0731676, 3.1413669] # This is the base turn 

